### PR TITLE
Ignore creating client with token if local server is used

### PR DESCRIPTION
**Pull request by juandspy**

### CODE REVIEW

File: rhoas/provider.go [hunk 1]
```diff
@@ -93,11 +94,14 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 		os.Exit(1)
 	}
 
-	// nolint: contextcheck
-	httpClient := authAPI.BuildAuthenticatedHTTPClient(d.Get("offline_token").(string))
-
 	localDevelopmentServer := os.Getenv(LocalDevelopmentEnv)
 
+	httpClient := &http.Client{}
+	if localDevelopmentServer == "" {
```
Comment (dimakis) [inline rhoas/provider.go:100]: LGTM it is exactly what I'm doing with Ansible.
I'm unsure if `""` will suffice? For Ansible the variable is `None` type if not passed through. I'm not sure how TF works so you may be right. just something to keep an eye out for

Comment (juandspy) [reply rhoas/provider.go:100]: This is Go, here there is no `None` :) Also, as `localDevelopmentServer` is a string, `""` would be the `nil` value.

**Comment by jackdelahunt** [comment]
LGTM 👍

**Comment by jackdelahunt** [comment]
@juandspy please do not request reviews from pmuir as he is no longer active on the project... unless if he wants to of course 😄

**Pull request merged**